libc/src/__support/time/linux/timeout.h

```diff
@@ -0,0 +1,60 @@
+//===--- timeout linux implementation ---------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_LIBC_SRC___SUPPORT_TIME_LINUX_TIMEOUT_H
+#define LLVM_LIBC_SRC___SUPPORT_TIME_LINUX_TIMEOUT_H
+#include "hdr/time_macros.h"
```

Comment:
nit: Do you mind adding some extra lines to separate the `#include`'s from the `#define` and `namespace`?

Reply:
sure. fixed in [623352b](https://github.com/llvm/llvm-project/pull/91905/commits/623352b426884f1d672fb4a3fc772113a82a1e51)